velox/common/base/SimdUtil-inl.h

```diff
@@ -1436,4 +1436,189 @@ inline bool memEqualUnsafe(const void* x, const void* y, int32_t size) {
   return true;
 }
 
+namespace detail {
+template <typename T>
+T clearLeftmostSet(const T value) {
```

Comment:
These are well known patterns, just use them directly without helper functions.  Giving them names is rather confusing (first vs last vs left vs right)